src/patternfly/components/Table/table-grid.scss

```diff
@@ -43,14 +43,14 @@
   // Body
   --pf-c-table-tbody--responsive--MarginTop: var(--pf-global--spacer--xs);
   --pf-c-table-tbody--m-expanded--before--Top: var(--pf-global--spacer--sm);
-  --pf-c-table-tbody--responsive--BorderWidth: var(--pf-global--spacer--sm);
-  --pf-c-table--tbody--after--BorderWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-table-tbody--responsive--border-width--base: var(--pf-global--spacer--sm);
```

Comment:
This var is `--pf-c-table-tbody....` but the ones below are `--pf-c-table--tbody...`